sa/sa_test.go

```diff
@@ -97,12 +93,12 @@ func TestAddRegistration(t *testing.T) {
 	test.AssertEquals(t, dbReg.ID, expectedReg.ID)
 	test.Assert(t, core.KeyDigestEquals(dbReg.Key, expectedReg.Key), "Stored key != expected")
 
-	u, _ := core.ParseAcmeURL("test.com")
+	u := "test.com"
```

Comment:
Nit: could probably drop `u` here now that there's no err return to discard and just use `Contact: &[]string{"test.com"},` on Line 101.